ingest/src/main/java/com/connexta/ingest/IngestApplication.java

```diff
@@ -1,18 +1,9 @@
 /*
- * Copyright (c) Connexta
- *
- * This is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as
- * published by the Free Software Foundation, either version 3
- * of the License, or any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Lesser General Public License for more details. A copy of the
- * GNU Lesser General Public License is distributed along with this
- * program and can be found at http://www.gnu.org/licenses/lgpl.html.
- */
+Copyright (c) 2019 Connexta, LLC
```

Comment:
❓ Does the license need to be on non-java files too?

Reply:
We haven't applied the license to non-java files so far. The guidance I'm going off of says all source-controlled files, so I would think non-java files apply.

Reply:
I'm fine addressing the non-java files in another PR if you can open an issue against this repo to do that later.

Reply:
https://github.com/connexta/multi-int-store/issues/26